x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -355,6 +356,16 @@ export interface GetExecutionLogByIdParams {
   sort: estypes.Sort;
 }
 
+export interface GetActionErrorLogByIdParams {
+  id: string;
+  dateStart: string;
+  dateEnd?: string;
+  filter?: string;
+  page: number;
+  perPage: number;
+  sort: FindOptionsType['sort'];
```

Comment:
Do you think it would make sense for the sort field be passed in as an `estypes.Sort` like the executuion log parameters and then for the `getActionErrorLogs()` function to reformat it into what the event log client is expecting? That would make the HTTP API definitions for these two functions consistent.

Reply:
We could even add a function to the event log to convert from ES sort to event log format and perhaps that can be reused in the future.

Reply:
yep sounds good!